### PR TITLE
FEATURE: QA-17837  Migrate Attribute filter tests from Graphene to SDK

### PR DESCRIPTION
**Pull request by NghiLamGD**
---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10928/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13022/ : FAILURE in 7m 44s

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardFiltering.spec.ts [hunk 1]
```diff
@@ -97,4 +108,213 @@ describe.skip("Dashboard Filtering", { tags: ["pre-merge_isolated_bear"] }, () =
         cy.wait(200);
         headline.waitLoaded().hasValue("$6,111,808.02");
     });
+
+    it("Update values on edit mode after making change on view mode", () => {
+        Navigation.visit("dashboard/attribute-filtering");
```
[the remark this answers is not in the document]
Comment (NghiLamGD) [reply libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardFiltering.spec.ts:115]: imo, no need to do this, this string is already defined in ComponentResolver

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by hungnguyenauto — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by no23reason** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10941/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/345/ : FAILURE in 5m 07s

**Comment by no23reason** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10942/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13038/ : FAILURE in 8m 46s

**Comment by no23reason** [comment]
extended check cypress

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10949/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/347/ : FAILURE in 5m 07s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10948/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13045/ : FAILURE in 8m 25s

**Comment by gdgate** [comment]
Build in pipeline check-extended-cypress aborted.

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10953/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13049/ : SUCCESS in 9m 55s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10954/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/349/ : FAILURE in 18m 06s

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10956/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/350/ : FAILURE in 19m 21s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10959/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13054/ : SUCCESS in 11m 17s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10960/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/351/ : FAILURE in 17m 56s

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10961/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13055/ : SUCCESS in 11m 09s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10962/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/352/ : FAILURE in 17m 32s

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10963/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13056/ : SUCCESS in 9m 03s

**Comment by gdgate** [comment]
Build in pipeline check-extended-cypress aborted.

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10965/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13057/ : SUCCESS in 10m 28s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardFiltering.spec.ts**|18(3 skipped)|-|133.4350s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|19.5510s|
|✅|**barChartDrilling.spec.ts**|4|-|6.2960s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|38.3290s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|6.5760s|
|✅|**attributeFilterButton.spec.ts**|9|-|29.7440s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|7.1290s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|6.5080s|
|✅|**headlineChartDrilling.spec.ts**|2|-|3.7240s|
|✅|**dateFilter.spec.ts**|9|-|14.0730s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|16.6960s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|13.7690s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|6.5800s|
|✅|**pivotTableSizing.spec.ts**|1|-|4.4120s|
|✅|**dashboard.spec.ts**|9|-|61.7360s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/354/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10966/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/354/ : SUCCESS in 16m 06s

**Comment by NghiLamGD** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10971/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13060/ : SUCCESS in 9m 27s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**pivotTableSizingReset.spec.ts**|15|-|40.4700s|
|✅|**dateFilter.spec.ts**|9|-|14.3480s|
|✅|**barChartDrilling.spec.ts**|4|-|6.5950s|
|✅|**pivotTableSizing.spec.ts**|1|-|5.1540s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|7.2390s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|14.4710s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|20.6460s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|151.8000s|
|✅|**dashboard.spec.ts**|9|-|57.4830s|
|✅|**attributeFilterButton.spec.ts**|9|-|30.7610s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|7.1130s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|7.4280s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|17.2890s|
|✅|**headlineChartDrilling.spec.ts**|2|-|4.1700s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|7.2120s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/357/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10972/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/357/ : SUCCESS in 14m 55s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3354

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/228/ : FAILURE in 2m 28s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3494/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2343/ : SUCCESS in 15m 50s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13061/ : SUCCESS in 12m 25s

**Pull request merged**